grails-app/domain/org/pih/warehouse/order/OrderAdjustment.groovy

```diff
@@ -109,7 +109,7 @@ class OrderAdjustment implements Serializable, Comparable<OrderAdjustment> {
     }
 
     Boolean isInvoiceable() {
-        return !isInvoiced && order.placed
+        return hasPrepaymentInvoice && !hasRegularInvoice && order.placed
```

Comment:
Is this because `isInvoiced` used for non-prepay flows?

Reply:
`isInvoiced` looks for posted invoices, and here we want to have both posted and not yet posted invoices